.github/workflows/ci-backend.yml

```diff
@@ -9,6 +9,9 @@ on:
         branches:
             - master
     pull_request:
+        paths-ignore:
```

Comment:
this doesn't work sadly because these checks are required, and required checks are marked skipped if we hit the `paths-ignore` field. This is why we have the dorny path filter annoyance below

Reply:
(it's running on this PR because this CI file changed :P )